src/jwt.rs

```diff
@@ -1,12 +1,7 @@
-extern crate base64;
 use base64::decode;
-
-extern crate serde;
 use serde::{Deserialize, Serialize};
```

Comment:
Is there convention or holy war around importing libraries or specific items in libraries?

Reply:
There are different styles, but there's no holy war (yet).  Most code I've seen imports specific items, except where there is ambiguity (eg: `io::Result`).  Note you can also use 'self' to import both in one statement: `use serde::{self, Deserialize};` is the same as `use serde; use serde::Deserialize;`.

`rustfmt` is fairly new, and I expect "whatever it does" will become the one-true-answer, as the golang precedent has demonstrated.  By corollary, "whatever it _doesn't_ enforce" is a legitimate local style choice. I expect on this question, rustfmt will never mechanically force an answer, and thus both variants will always be acceptable.

Just to mess it up further, there's also nested imports now: https://doc.rust-lang.org/edition-guide/rust-2018/module-system/nested-imports-with-use.html